src/components/Header/Header.js

```diff
@@ -5,7 +5,7 @@ import styles from './Header.module.css';
 import logo from './logo.png';
 import {ImInfo} from "react-icons/im";
 import {RiTeamLine, RiContactsBook2Line} from "react-icons/ri";
-import {BiDonateHeart, BiMenu} from "react-icons/bi";
```

Comment:
Can you please explain the reason behind this change?

Reply:
Sir in tailwind there is no need for hamburger as it itself gets responsive so I made this changes.

Reply:
What about the animation?😅